src/db/OrganizationSchema.ts

```diff
@@ -0,0 +1,69 @@
+import mongoose from 'mongoose'
+import muuid from 'uuid-mongodb'
+
+import { OrganizationType, OrgType, IOrganizationContent, OperationType } from './OrganizationTypes.js'
+import { ChangeRecordMetadataType } from './ChangeLogType.js'
+
+const { Schema, connection } = mongoose
+
+const ChangeRecordMetadata = new Schema<ChangeRecordMetadataType>({
+  user: {
+    type: 'object',
+    value: { type: 'Buffer' },
+    required: true
+  },
+  historyId: { type: Schema.Types.ObjectId, ref: 'change_logs' },
+  operation: {
+    type: Schema.Types.Mixed,
+    enum: Object.values(OperationType),
+    required: true
+  },
+  seq: { type: Number, required: true, default: 0 }
+}, { _id: false, timestamps: false })
+
+const ContentSchema = new Schema<IOrganizationContent>({
```

Comment:
Not sure if splitting out to a sub-schema has any value. But we do it quite liberally in the AreaSchema and elsewhere. Eg. `area.metadata`, `area.content`. I would be interested to understand the rationale for when to break things out and when not to!